rgw/v2/tests/s3_swift/test_multisite_bucket_granular_sync_policy.py

```diff
@@ -270,9 +278,92 @@ def test_exec(config, ssh_con):
+                                    bkt.name, oc
+                                )
+                                log.info(f"s3 object name: {s3_object_name}")
+                                s3_object_path = os.path.join(
+                                    TEST_DATA_PATH, s3_object_name
+                                )
+                                log.info(f"s3 object path: {s3_object_path}")
+                                log.info("upload type: normal")
+                                reusable.upload_object(
+                                    s3_object_name,
+                                    other_site_bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+
+                            _, stats_stdout, _ = rgw_ssh_con.exec_command(
+                                f"radosgw-admin bucket stats --bucket {bkt.name}"
+                            )
+                            re_cmd_output = json.loads(stats_stdout.read().decode())
+                            log.info(f"re_cmd_output : {re_cmd_output}")
+                            new_object_count = num_objects + config.objects_count
+                            if (
+                                re_cmd_output["usage"]["rgw.main"]["num_objects"]
+                                != new_object_count
+                            ):
+                                raise TestExecError(
+                                    f"Failed to upload object to bucket {bkt.name}"
+                                )
+
+                            log.info(
+                                f"Verify object sync on other site for bucket {bkt.name}"
+                            )
+                            time.sleep(1200)
+                            bucket_stats = json.loads(
+                                utils.exec_shell_cmd(
+                                    f"radosgw-admin bucket stats --bucket {bkt.name}"
+                                )
+                            )
+                            bkt_objects = bucket_stats["usage"]["rgw.main"][
+                                "num_objects"
+                            ]
+
+                            if config.test_ops.get(
+                                "write_io_verify_should_sync", False
+                            ):
+                                if bkt_objects != config.objects_count * 2:
+                                    raise TestExecError(
+                                        f"Object did not sync in bucket {bkt.name}, but found {bkt_objects}"
+                                    )
+                                log.info(
+                                    f"Object synced for bucket {bkt.name}, on another site as expected"
+                                )
+                            else:
+                                if bkt_objects == config.objects_count * 2:
```

Comment:
@anrao19 , could you please change the condition to `if bkt_objects != config.objects_count:`
instead of particularly checking if equals to double of object count, this conditional covers the other corner cases as well right